examples/md-flexible/src/Simulation.h

```diff
@@ -251,7 +246,7 @@ class Simulation {
   /**
    * Updates the position of particles in the local AutoPas container.
```

Comment:
Modify this comment

Reply:
done in [d6a43cc](https://github.com/AutoPas/AutoPas/pull/905/commits/d6a43cca1508fbfd43a46ec8ed39b59d55e687aa)